.pre-commit-hooks.yaml

```diff
@@ -18,3 +18,31 @@
   language: python
   types: [shell]
   args: [--fix]
+- id: verify-copyright
+  name: copyright headers
+  description: make sure copyright headers are up to date
+  entry: verify-copyright
+  language: python
+  files: |
+    (?x)
+        [.](cmake|cpp|cu|cuh|h|hpp|sh|pxd|py|pyx)$|
+        CMakeLists[.]txt$|
+        CMakeLists_standalone[.]txt$|
+        meta[.]yaml$|
+        pyproject[.]toml$|
+        setup[.]cfg$
+  args: [--fix]
+- id: verify-copyright-batch
```

Comment:
The idea is that the `verify-copyright-batch` hook would be used on a one-time basis to bring a repo up to date (perhaps with `pre-commit try-repo -a verify-copyright-batch`), and from then on it would only use the `verify-copyright` hook. This seems more intuitive to me than changing the arguments for an existing hook. WDYT?

Reply:
I agree with how you would use it (`--batch` first, then no need for `--batch` anymore). I think it's equally to tell users to add `args: ["--fix", "--batch"]` as to use a different hook name for the initial run. I was hoping to remove the duplicated logic (especially for the list of `files:`) between `verify-copyright` and `verify-copyright-batch` so we don't have to keep them both up to date (especially for a batch mode that will be rarely invoked).

Reply:
You make a good point with the duplicated `files:`. I've removed the duplicate hook - people can add the `--batch` argument instead.